generators/opa5/index.js

```diff
@@ -1,159 +1,66 @@
-import Generator from "yeoman-generator";
-import path from "path";
-import glob from "glob";
-import { fileURLToPath } from 'url';
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = path.dirname(__filename);
+import chalk from "chalk"
+import fs from "fs"
+import Generator from "yeoman-generator"
+import prompts from "./prompts.js"
+import {
+	lookForParentUI5ProjectAndPrompt,
+	addPreviewMiddlewareTestConfig,
+	ensureCorrectDestinationPath
+} from "../helpers.js"
+import yaml from "yaml"
+import path, { dirname } from "path"
```

Comment:
might only import `join` and `dirname` eventually as you only seem to be using that from `path`

Reply:
Hm, I'd default to always importing `path` as it adds to readability when using `path.join()`, imo at least.